CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# v0.2.0
+### Improvements
+* Ability to re-generate secrets from .properties file
+* Group Gradle tasks and their descriptions to convenience command `tasks`
+#### Contributors
+@validya
```

Comment:
Just a minor typo, if I may, it's should be `@valydia`

Reply:
@valydia fixed